utils/check_repo.py

```diff
@@ -30,6 +30,7 @@
     "test_modeling_tf_xlm_roberta.py",
     "test_modeling_xlm_roberta.py",
     "test_modeling_pegasus.py",
+    "test_modeling_blenderbot.py",
```

Comment:
Blenderbot should have common tests. The model is different enough to Bart to have it's own tests.